spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -1,12 +1,13 @@
 ---
 ics: 20
 title: Fungible Token Transfer
+version: 2 
 stage: draft
 category: IBC/APP
 requires: 25, 26
 kind: instantiation
 version compatibility: ibc-go v7.0.0 (ics20-1 supported only)
-author: Christopher Goes <cwgoes@interchain.berlin>
+author: Christopher Goes <cwgoes@interchain.berlin>, Aditya Sripal <aditya@interchain.io>,
 created: 2019-07-15 
 modified: 2024-03-05
 ---
```

Comment:
Maybe we should have a note here or at the top which ibc-go/ibc-rs versions support v1 and which support v2